### ISSUE TITLE
UX: Simple design for direct message instead of fab

### ISSUE THREAD

**Issue by AnisTaluqdar** [syphon-org/syphon#506]
![IMG_20220121_153949.jpg](https://user-images.githubusercontent.com/41234183/150624990-0488adab-a478-41f6-b82e-41db51710513.jpg)

Already, in three do option there have 
[ create a group ]. just need to move the search group and create a public group.

And instead of fab, there can be only direct messages. It will be the more user-friendly and modern way.

**Comment by hollisticated-horse** [comment]
Hey @AnisTaluqdar, I'm a bit confused. Is the problem the many options to create a chat with others ? Or the too many buttons in the quick access button ?

**Comment by AnisTaluqdar** [comment]
> the too many buttons in the quick access button ?

Yes, it is. Simple thing always better for user. Most of the popular messaging app simple UI. And user like to use simple things.

**Comment by AnisTaluqdar** [comment]
In syphon everything is simple without this fab tab
